apps/emqx_conf/src/emqx_conf_cli.erl

```diff
@@ -290,23 +309,13 @@ load_etc_config_file() ->
             {error, Error}
     end.
 
-check_readonly_config(Raw) ->
+filter_readonly_config(Raw) ->
     SchemaMod = emqx_conf:schema_module(),
     RawDefault = emqx_config:fill_defaults(Raw),
     case emqx_conf:check_config(SchemaMod, RawDefault) of
-        {ok, CheckedConf} ->
-            case filter_changed_readonly_keys(CheckedConf) of
```

Comment:
before: 
the check will failed if  readonly key changed.
after: 
we just filter readonly key.